packages/terra-graphs-docs/CHANGELOG.md

```diff
@@ -66,3 +66,4 @@
   * Updated Combination graph documentation to include api information.
   * Updated Getting Started documentation to be consistent with the rest of the graph docs.
   * Updated example extensions from .doc to .graph.
+  * Added reflowMultipleDataset documentation in panning docs.
```

Comment:
terra-graphs-docs has been release so this should be moved to the `Unreleased` section.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/273/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R5).